### ISSUE TITLE
hash/anchor links break

### ISSUE THREAD

**Issue by dangayle** [dhakerShiv/gatsby-plugin-client-side-redirect#1]
In this code, there is a `/` added to the ${url}.

```js
    const resemblesFile = url.includes('.');
    if (!resemblesFile) {
      url = `${url}/`.replace(/\/\/+/g, '/');
    }
```

It breaks redirects that are based on hashes, like the following:

```js
createRedirect({ fromPath: '/reviews', toPath: '/#reviews', isPermanent: true });
```

**Comment by dhakerShiv** [comment]
It will be great if you can raise a PR for it :)

**Comment by NerdCowboy** [comment]
@dangayle @dhakerShiv #2 should fix this issue, so should be able to close

**Comment by dhakerShiv** [comment]
@dangayle  Issue has been fixed in #3 , so closing it :).
